addon/mixins/flexberry-map-actions-handler.js

```diff
@@ -0,0 +1,202 @@
+  actions: {
+    /**
+      Handles {{#crossLink "FlexberryMapComponent/sendingActions.leafletInit:method"}}flexberry-map component's 'leafletInit' action{{/crossLink}}.
+      It stores value of action's event object 'map' property in property with given path.
+
+      @method actions.onMapInit
+      @param {String} mutablePropertyPath Path to a property, which value must be mutated on action.
+      @param {Object} e Action's event object.
+      @param {Object} e.map Initialized [Leaflet map](http://leafletjs.com/reference-1.0.0.html#map).
+
+      @example
+      templates/my-form.hbs
+      ```handlebars
+        {{flexberry-map
+          lat=model.lat
+          lng=model.lng
+          zoom=model.zoom
+          leafletInit=(action "onMapLeafletInit" "leafletMap")
+        }}
+      ```
+
+      controllers/my-form.js
+      ```javascript
+        import Ember from 'ember';
+        import FlexberryMapActionsHandlerMixin from 'ember-flexberry-gis/mixins/flexberry-map-actions-handler';
+
+        export default Ember.Controller.extend(FlexberryMapActionsHandlerMixin, {
+          // Property where leaflet map will be stored after init.
+          leafletMap: null
+        });
+      ```
+    */
+    onMapLeafletInit(...args) {
+      // User can pass any additional arguments to action handler,
+      // so original action's event object will be the last one in arguments array.
+      let leafletMapPropertyPath = args[0];
+      let e = args[args.length - 1];
+
+      Ember.assert(
+        `Wrong type of \`leafletMapPropertyPath\` argument: actual type is \`${leafletMapPropertyPath}\`, ` +
+        `but \`string\` is expected`,
+        Ember.typeOf(leafletMapPropertyPath) === 'string');
+
+      this.set(leafletMapPropertyPath, e.map);
+    },
+
+    /**
+      Handles {{#crossLink "FlexberryMapComponent/sendingActions.leafletDestroy:method"}}flexberry-map component's 'leafletDestroy' action{{/crossLink}}.
+      It cleans up previously stored leaflet map.
+
+      @method actions.onMapLeafletDestroy
+      @param {String} leafletMapPropertyPath Path to a previously stored leaflet map, which must be cleaned up on action.
+
+      @example
+      templates/my-form.hbs
+      ```handlebars
+        {{flexberry-map
+          lat=model.lat
+          lng=model.lng
+          zoom=model.zoom
+          leafletInit=(action "onMapLeafletInit" "leafletMap")
+          leafletDestroy=(action "onMapLeafletDestroy" "leafletMap")
+        }}
+      ```
+
+      controllers/my-form.js
+      ```javascript
+        import Ember from 'ember';
+        import FlexberryMapActionsHandlerMixin from 'ember-flexberry-gis/mixins/flexberry-map-actions-handler';
+
+        export default Ember.Controller.extend(FlexberryMapActionsHandlerMixin, {
+          // Property where leaflet map will be stored after init.
+          leafletMap: null
+        });
+      ```
+    */
+    onMapLeafletDestroy(...args) {
+      let leafletMapPropertyPath = args[0];
```

Comment:
Ember action helper pass arguments array, not an object.
Code like "onMapLeafletDestory({ leafletMapProopertyPath })" will throw an exception.